src/sentry/plugins/sentry_mail/models.py

```diff
@@ -123,6 +126,43 @@ def get_send_to(self, project):
             logger.debug('Tried to send notification to invalid project: %r', project)
             return []
 
+        if event:
+            owners = ProjectOwnership.get_owners(project.id, event.data)
+            if owners != ProjectOwnership.Everyone:
+                if not owners:
+                    metrics.incr(
+                        'owners.empty.match',
+                        tags={'organization': project.organization_id},
+                        skip_internal=True,
+                    )
+                    return []
+
+                from sentry.models import User
+                metrics.incr(
+                    'owners.match',
+                    tags={'organization': project.organization_id},
+                    skip_internal=True,
+                )
+                send_to_list = []
+                teams_to_resolve = []
+                for owner in owners:
+                    if owner.type == User:
+                        send_to_list.append(owner.id)
+                    else:
+                        teams_to_resolve.append(owner.id)
+
+                # get all users in teams
+                if teams_to_resolve:
+                    send_to_list += User.objects.filter(
+                        sentry_orgmember_set__organizationmemberteam__team__id__in=teams_to_resolve,
```

Comment:
@mattrobenolt I believe this works, butttt... I'm sure I forgot things. Do I need to check if they are inactive; I think I probably should actually....? things like that? Right now I'm just grabbing all users in teams.

Reply:
Correct, add an `is_active=True` filter here.